src/components/Signup/Signup.js

```diff
@@ -97,6 +98,7 @@ const Signup = function ({
     UTMMedium: utmMedium,
     UTMTerm: utmTerm,
     gclid,
+    UTMContent: utmContent,
```

Comment:
agregarla en los utm cookie nosé si sea necesario

Reply:
Yes we do need this in utm cookies. By requirement.